index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Daniels Portfolio</title>
+</head>
+<body>
+    <nav>
+        <ul class="buger">
+
+            <li class="Mi">Daniel</li>
+           <li class="ham">
+            <i class="fa fa-bars"></i> 
+           </li>  
+        </ul>
+    </nav>
+    <section class="homesection">
+        <div>
+<h2 class="intro">I'm Daniel glad to see you</h2>
```

Comment:
Kindly, I suggest you use the `h1` tag instead of `h2` for the first heading to follow HTML best practices. 😄